tests/venv/py3/README.rst

```diff
@@ -4,6 +4,5 @@ System - Specifying runtime environments
 You can specify runtime environments (such as Python 2 or 3) with a
 ``runtime.txt`` file.
 
-This is an example that selects Python 3. Currently you can not use
-this to select a sepcific version of Python 3 (e.g. 3.4 vs 3.6). If you
-need this level of control we recommend you use a `environment.yml`.
+This is an example that selects Python 3.
+If you need this level of control we recommend you use a `environment.yml`.
```

Comment:
Perhaps there's something deleted that shouldn't be? It sounds like this line is suggesting that people just don't ever use `runtime.txt`